tools/myst/host/syscall.c

```diff
@@ -524,8 +524,8 @@ long myst_fsync_ocall(int fd)
 long myst_get_mempolicy_ocall(
     int* mode,
     unsigned long* nodemask,
+    size_t nodemask_size,
```

Comment:
`maxnode` parameter one line 532 can be derived from `nodemask_size`, no need to include 'maxnode` as an OCALL parameter.

Reply:
since nodemask_size is only needed for the myst.edl, I like the idea of keeping the parameters separate from each other. In the event I take out maxnode in the OCALL implementation, there would be double computation of first calculating nodemask_size from maxnode (maxnode/8) and then calculating maxnode from nodemask size, which seems unideal.

Reply:
Added comment & void statement